recipes/feather-format/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.7
```

Comment:
If 10.7 is the earliest deployment target supporting C++11 we should update the docs in the Feather repo too